src/csharp/Grpc.Examples/MathGrpc.cs

```diff
@@ -67,6 +67,7 @@ public static partial class Math
     }
 
     /// <summary>Base class for server-side implementations of Math</summary>
+    [grpc::BindService(typeof(Math), "BindService")]
```

Comment:
nit: TBH i'm still not 100% convinced by the name `[BindService]`, I don't feel the name fully conveys what the attribute stands for. Have we considered some alternative names?

Reply:
I'm open to alternatives. `ServiceBindAttribute`? `ServiceBinderAttribute`? `ServiceBindMethodAttribute`?